### PR TITLE
Basic structure branch

### PR DESCRIPTION
**Pull request by Gambit142**
In this branch, I did the following:

- I made sure that there are no linter errors.
- I made sure that I used the correct Gitflow.
- I made sure that I documented your work in a professional way.
- I used HTML markup with minimum styling to build a medium wireframe website.
- I used ES6 modules, with import and export.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,46 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Leaderboard
+> This website creates a working version of the leaderboard that preserves user input. The website is built using medium wireframe
 
-> One paragraph statement about the project.
+> |Mobile Version|Desktop Version
+> |--------------|--------------|
+> |![screenshot](Leaderboard-mobile.png).|![screenshot2](LeaderBoard-desktop.png)|
```
Comment (helman101) [inline README.md]: - [ ] The desktop version screenshot is not working, kindly fix this issue, make sure that you are using the right filename.
    - I would recommend moving the screenshots to another folder 😄

File: src/index.js [hunk 2]
```diff
@@ -1,11 +1 @@
 import './style.css';
```
Comment (helman101) [inline src/index.js:1]: - [ ] At this point of the project you should have developed a working version of the application as the project requirements said. Kindly take a look at this screenshot and to project requirements. 😸 
    - All the buttons should be functional, which means, the submit button should add a score to the list, and the refresh button should refresh the list of scores 🚀
    - Remember to use ES6 syntax
    imagen

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🔴 

Hi @Gambit142  👋, 

You have done a great job with the template of your page,  👏👏👏 
but there are still some changes that you have to do.

## To Highlight 🌟 

- Nice job on the page template 😄 
- Nice job setting up webpack 🚀 

## Required Changes ♻️

- [ ] If you want to use an MIT license please make sure that you have replaced the [YOUR NAME] and [APP TYPE] spaces with your information. 😄 
     - [APP TYPE] could be changed for `web application` or `web page`
- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (Gambit142) [reply src/index.js:1]: Hi, thank you for the feedback.
this is the first milestone for the entire project and it says we should just add basic `HTML` markup and little styling. It doesn't state that we should add functionality to the button yet.

Kindly find the link to the project requirement for the first milestone below:
https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md

File: README.md (hunk 1 above)
Comment (Gambit142) [reply README.md]: Thank you for bringing my attention to this, I have corrected it and it displays properly. 

I have also made changes to the `MIT.md` file and filled the necessary spaces with my name and the name of this repository.

Thank you once again.

**Review by carlos-ssh — APPROVED** [review]
Hi @Gambit142 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gambit142** [comment]
Thank you for the feedback

**Pull request merged**